contrib/sssd.spec.in

```diff
@@ -1061,6 +1058,10 @@ getent passwd sssd >/dev/null || useradd -r -g sssd -d / -s /sbin/nologin -c "Us
 %systemd_preun sssd-sudo.socket
 
 %postun common
+rm -f %{mcpath}/passwd
+rm -f %{mcpath}/group
+rm -f %{mcpath}/initgroups
+rm -f %{mcpath}/sid
```

Comment:
%__rm should be used instead and `coreuitls` should be required as `Requires(postun)`